### ISSUE TITLE
[Programmatic Access – Make Code - Explorer] Incorrect role is defined as 'Tree item' for 'Explorer button and their menu items' present on 'Editor' screen.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3641]
**User Experience:**
Users who depends upon screen reader get confused if the Screen reader reads incorrect role of any control. They'll not be able to know whether it is a link or a button or whether it is a non-interactive control. Users will understand it to be a interactive control as screen reader announces link while navigating in scan mode and hence it will not be a good usable experience for the users. 

**Note:** User credentials should NOT be included in the bug. 

**Pre-Requisite:**
Open F12 Developer tool.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Java Script" drop down and select it.
6. Navigate to "Python" option and select it.
7. Navigate to "Explorer" and select it.
8. Navigate through all the controls after expanding Explorer control.
9. Verify that incorrect role is defined for 'Explorer button and their menu items'.

**Actual Result:**
Incorrect role is defined as 'Tree item' for 'Explorer button and their menu items' present on 'Editor' screen.

**Expected Result:**
Correct role should be define as 'Button' instead of 'tree item' for 'Explorer button' and 'Menu item' for 'Main.blocks, Main.ts, pxt.json, README.md, Core, Radio, Built' controls present on 'Explorer' button.

**Note:**
Same issue exists on Firefox + NVDA & Chrome + JAWS.

**WCAG Reference:**
[WCAG 4.1.2 – Name, Role, Value ](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.1.2, #Win10-Edge(Chromium)

![WCAG4 1 2_Explorer_Incorrect role](https://user-images.githubusercontent.com/72871089/98653956-ae6c4e00-2363-11eb-8ab5-160399e64693.jpg)

**Comment by jwunderl** [comment]
These elements are part of a tree menu (/ leaf nodes on a tree menu) where folders can be expanded / collapsed
